ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1296,6 +1287,8 @@ namespace OrdinalAxis {
                     fakeAxis.series.push(fakeSeries);
 
                     series.processData.apply(fakeSeries);
+
+                    fakeSeries.applyGrouping.apply(fakeSeries);
```

Comment:
Waaait. The idea with `Axis.applyGrouping` (the method that calls `Series.applyGrouping`) is to use this method **after** each series has been processed (after `processData()`). Could you try it?